package.json

```diff
@@ -32,9 +32,6 @@
     "dependencies": {
         "graphql": "^14.7.0",
         "json-to-graphql-query": "^1.9.0",
-        "lodash.isarray": "^4.0.0",
-        "lodash.isobject": "^3.0.2",
-        "lodash.isstring": "^4.0.1",
         "lodash.mapvalues": "^4.6.0"
```

Comment:
Lodash mapvalues is tricker to replace with native JS. We should look to replace this in the future